### ISSUE TITLE
Do not use Number instead of Size

### ISSUE THREAD

**Issue by olexandr-konovalov** [gap-packages/RAMEGA#4]
For example, in:

https://github.com/gap-packages/RAMEGA/blob/c1e2a4dfab15f74fb176d5975da4b39c4372ca9a/lib/internal.gi#L99

and perhaps you use it elsewhere - needs to be fixed throughout.

Size(G) returns it order, which is an attribute, in most cases known or easy to compute once (since it's an attribute). `Number` iterates over the group - see its documentation. So, for example:
```
gap> G:=MathieuGroup(23);
Group([ (1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23), 
  (3,17,10,7,9)(4,13,14,19,5)(8,18,11,12,23)(15,20,22,21,16) ])
gap> Size(G);
10200960
gap> time;
0
gap> Number(G);
10200960
gap> time;
12664
```
so almost 13 seconds instead of zero.

**Comment by vlaver** [comment]
> Size(G) returns it order, which is an attribute, in most cases known or easy to compute once (since it's an attribute). `Number` iterates over the group - see its documentation. 

Fixed.

**Comment by olexandr-konovalov** [comment]
Thanks. BTW you can write e.g. `Closes #4` in line 3 or below of your commit message so that the issue will be closed automatically (see https://help.github.com/en/enterprise/2.16/user/github/managing-your-work-on-github/closing-issues-using-keywords for all recognised keywords).

Closed by b2afa32923701cbfd843d61c663f39aed946149a.
